examples/pytorch/barlowtwins.py

```diff
@@ -0,0 +1,61 @@
+import torch
+from torch import nn
+import torchvision
+
+import lightly
+from lightly.models.modules import BarlowTwinsProjectionHead
```

Comment:
It's a little strange that you import this directly but use `lightly.loss.BarlowTwins` and `lightly.data.LightlyDataset`

Reply:
Updated :)